src/Persistent_cohomology/include/gudhi/Persistent_cohomology.h

```diff
@@ -600,8 +600,10 @@ class Persistent_cohomology {
    * @return A vector of Betti numbers.
    */
   std::vector<int> betti_numbers() const {
+    // Don't allocate a vector of negative size for an empty complex
+    int siz = std::max(dim_max_, 0);
     // Init Betti numbers vector with zeros until Simplicial complex dimension
-    std::vector<int> betti_numbers(dim_max_, 0);
+    std::vector<int> betti_numbers(siz);
```

Comment:
Init the vector with zeros, or the comment needs to be updated.

Reply:
The `0` is redundant, that's the default behavior.